tests/verify_aaefold_nval_api.erl

```diff
@@ -233,4 +336,26 @@ wait_until_root_stable(Mod, Client) ->
             lager:info("Root not stable: branch ~w compares ~w with ~w",
                         [L, V0, V1]),
             wait_until_root_stable(Mod, Client)
-    end.
\ No newline at end of file
+    end.
+
+wait_until_object_format(Nodes, Format) ->
```

Comment:
confusing that you call this with Format `v1` in line 254
and you expect the RPC with get for `v0` respond with v1??